### ISSUE TITLE
Auto-post to Twitter

### ISSUE THREAD

**Issue by wilzbach** [CyberShadow/DFeed#63]
Automatize tweeting every announcement - maybe even with the official DLanguage account?

**Comment by CyberShadow** [comment]
This and #64 are only reasonable if it's done with a fresh new group / account, otherwise all actually important announcements and discussion will get lost in the noise of the D forum announcements. But then, would anyone follow it?

See the d_language subreddit for an example of what I meant. Right now it's completely dead, killed by spam from a bot that reposts all announcements there. Although people said they liked it, it killed nearly all participation in the subreddit.

**Comment by CyberShadow** [comment]
This can be done with no code written:

https://www.google.com/search?q=rss+to+twitter

https://www.google.com/search?q=rss+to+facebook

**Comment by wilzbach** [comment]
> See the d_language subreddit for an example of what I mean. Right now it's completely dead, killed by spam from a bot that reposts all announcements there. Although people said they liked it, it killed nearly all participation in the subreddit.

Fair enough, it seems pretty empty there (even though there are people liking those announcements). Maybe the better way is then to provide custom share buttons, so that the user can share his announcement easily if he wants to and then also is in charge himself of managing replies? Another plus of the share button would be that they get reminded about sharing it - which we also could put in the ui when one creates a new announcement thread?

**Comment by CyberShadow** [comment]
Share buttons is a possible idea. As long as it's not done by embedding other websites' widgets, as is popular. Perhaps a "share" action on the first post of every thread (like the "Subscribe" link), which opens a page or pop-up with the options to share it on Reddit / Facebook / etc.
